src/mixtures/unigmm.jl

```diff
@@ -1,20 +1,22 @@
 # Univariate Gaussian Mixture Models
 
-struct UnivariateGMM <: UnivariateMixture{Continuous,Normal}
+struct UnivariateGMM{T<:Real} <: UnivariateMixture{Continuous,Normal}
```

Comment:
```suggestion
struct UnivariateGMM{VT1<:AbstractVector{<:Real},VT2<:AbstractVector{<:Real}} <: UnivariateMixture{Continuous,Normal}
means::VT1
stds::VT2
```

This also allows for mean and std to have different element types (see the discussion in https://github.com/JuliaStats/Distributions.jl/pull/837 for motivation)